x-pack/test/functional/apps/lens/smokescreen.ts

```diff
@@ -26,12 +26,12 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
   const filterBar = getService('filterBar');
 
-  async function assertExpectedMetric() {
+  async function assertExpectedMetric(metricCount: string) {
```

Comment:
You could leave this with a default value, since you're testing the filtering only a few cases